packages/app/src/cli/models/extensions/specifications/ui_extension.test.ts

```diff
@@ -28,6 +28,9 @@ describe('ui_extension', async () => {
         block_progress: false,
         network_access: false,
         api_access: false,
+        collect_buyer_consent: {
+          sms_marketing: false,
```

Comment:
I'm not sure how useful it would be, but we don't have a test case that uses this capability when `sms_marketing: true`. D'you think it's worth adding one in one of these test files?